plugins/plugin-site/src/components/PluginReleases.jsx

```diff
@@ -9,6 +9,11 @@ import {formatter} from '../commons/helper';
 
 import './PluginReleases.css';
 
+function stripTag(tagName, pluginId) {
+    // assume tags are in the form ${pluginId}${infix}${version} or ${prefix}${version}
+    // where pluginId may contain digits, but prefix and infix cannot and version always starts with a digit
+    return tagName.replace(pluginId, '').replace(/^[^0-9]*/, '');
```

Comment:
I would have added a specific character as prefix to the tagName (like "[" for example), then:
```
return prefixedTagName.replace('[' + pluginId, '').replace(/^[^0-9]*/, '');
```
This would prevent version "number" to be replaced if they contain the plugin name